docs/service-introspection-onboarding.md

```diff
@@ -25,6 +45,115 @@ following properties of this storage before proceeding:
 Once you have a storage account with a table, you may proceed to start updating
 the pipelines to send data to Spektate storage.
 
+**Note:** The Azure storage account is needed to store information about your
+pipelines and services that is displayed by service introspection.
+
+2. Create a table. Follow these
+   [instructions](https://docs.microsoft.com/en-us/azure/storage/tables/table-storage-quickstart-portal).
+
+### Pipelines Configuration
+
+The Bedrock GitOps pipelines need to be configured to start sending data to
+`spk` service introspection. This is done by adding a script snippet in each
+`azure-pipelines.yml` configuration.
+
+#### Configure a variable group
+
+To send data from Azure pipelines to the service introspection storage created
+previously a variable group needs to be configured in Azure DevOps (where the
+pipelines are).
+
+To configure the variable group run: `spk variable-group create`
+
+#### Pipeline 1: CI pipeline configuration
+
+The CI pipeline runs from the source repository to build a docker image.
+
```

Comment:
Need a step for the use to add a variable group reference like  below. 
Make sure that you update the pipelines in the following steps to include this variable group, such as below:

variables:
  - group: <your-variable-group-name from above>